### PR TITLE
Feat/added navheader

### PR DESCRIPTION
**Pull request by ferreroltd**

### CODE REVIEW

File: packages/apps/dev-wallet/src/App/layout.tsx [hunk 1]
```diff
@@ -1,27 +1,54 @@
...
+  NavHeaderLink,
+  NavHeaderLinkList,
+  NavHeaderSelect,
+  SelectItem,
+  SystemIcon,
+} from '@kadena/react-ui';
+import { FC, useState } from 'react';
+import { Outlet } from 'react-router-dom';
+
+const mockNetworkItems: string[] = ['DevNet', 'Testnet', 'Mainnet'];
```
Comment (javadkh2) [inline packages/apps/dev-wallet/src/App/layout.tsx:13]: you can use `useNetwork` for the list of networks

File: packages/apps/dev-wallet/src/App/layout.tsx [hunk 2]
```diff
@@ -1,27 +1,54 @@
...
+        }
       >
-        <Stack gap="sm" padding="sm">
-          <Link to="/">
-            <Text bold>DX-Wallet</Text>
-          </Link>
-        </Stack>
-      </Stack>
+        <NavHeaderLinkList>
+          <NavHeaderLink key="home" href="/">
```
Comment (javadkh2) [inline packages/apps/dev-wallet/src/App/layout.tsx:28]: there are hard link. we need to use `react-router` links instead

File: packages/apps/dev-wallet/src/App/layout.tsx [hunk 3]
```diff
@@ -1,27 +1,62 @@
...
+  SystemIcon,
+  Text,
+} from '@kadena/react-ui';
+import { FC, useState } from 'react';
 import { Link, Outlet } from 'react-router-dom';
-import { layout } from './layout.css';
 
 export const Layout: FC = () => {
+  const { networks, activeNetwork, updateNetwork } = useNetwork();
+  const [value, setValue] = useState(activeNetwork?.networkId);
```
Comment (javadkh2) [inline packages/apps/dev-wallet/src/App/layout.tsx:16]: why do we need this state
cant we just use `activeNetwork.networkId`?

File: packages/apps/dev-wallet/src/App/layout.tsx [hunk 4]
```diff
@@ -1,27 +1,62 @@
...
-        </Stack>
-      </Stack>
+          <Link to="/networks">
+            <Text bold>Network</Text>
+          </Link>
+        </NavHeaderLinkList>
+
+        <NavHeaderSelect
+          aria-label="Select Network"
+          selectedKey={value}
```
Comment (javadkh2) [inline packages/apps/dev-wallet/src/App/layout.tsx:46]:
```suggestion
          selectedKey={activeNetwork.networkId}
```

File: packages/apps/dev-wallet/src/App/layout.tsx [hunk 5]
```diff
@@ -1,27 +1,62 @@
...
+import { FC, useState } from 'react';
 import { Link, Outlet } from 'react-router-dom';
-import { layout } from './layout.css';
 
 export const Layout: FC = () => {
+  const { networks, activeNetwork, updateNetwork } = useNetwork();
+  const [value, setValue] = useState(activeNetwork?.networkId);
+
+  const handleNetworkUpdate = (value: string) => {
+    setValue(value);
```
Comment (javadkh2) [inline packages/apps/dev-wallet/src/App/layout.tsx:19]:
```suggestion
```

File: packages/apps/dev-wallet/src/App/layout.tsx (hunk 3 above)
Comment (ferreroltd) [reply packages/apps/dev-wallet/src/App/layout.tsx:16]: yes will clear this indeed is redundant

**Review by javadkh2 — APPROVED** [review]

**Pull request merged**